⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim041
⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim042
⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim043
⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim044
⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim045
⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim046
⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim047
⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim048
⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim049
⚠️ 目標已存在: /home/aronton/tSDRG_random/tSDRG/Main_15/data_tar/PBC/Jdis060/Dim050
Traceback (most recent call last):
  File "/home/aronton/tSDRG_random/Subpy/copy2.py", line 23, in <module>
    shutil.copytree(src, dst)
  File "/home/aronton/anaconda3/lib/python3.12/shutil.py", line 598, in copytree
    with os.scandir(src) as itr:
         ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/homeold/aronton/tSDRG_random/tSDRG/Main_15/data_random/PBC/Jdis060/Dim051'
